### ISSUE TITLE
Generator PPA and Supplier 

### ISSUE THREAD

**Issue by GillianEnergyLocal** [marygillie/Advisor-club-development-queries#49]
Does the generator's PPA need to be with the same company that they pay their bills to? 

Eg. If a primary school has PV but the council arranges the procurement of utilities as part of a larger portfolio, would the school be able to join the club as a generator just by moving the PPA to Octopus?

**Comment by ghost** [comment]
Hi Gillian – you need to assign Mary in the Github. I have done this for you.

 

Have you asked Mary to make you a participant yet?  She needs to approve you 😊

 

Linda

**Comment by marygillie** [comment]
Yes a primary school or other council building can opt out of a big contract if they wish.  For some it is easier than others depending on their autonomy.
